android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java

```diff
@@ -128,6 +141,16 @@ public void onClick(View v) {
             snackbarText.setTextColor(Color.WHITE);
         }
 
+        if (RTL) {
+            snackbarView.setLayoutDirection(View.LAYOUT_DIRECTION_RTL);
+            snackbarView.setTextDirection(View.TEXT_DIRECTION_RTL);
```

Comment:
These both require API 17, and the min is currently 16. We should check for this in the condition above.

Reply:
I added new condition to check this